test/spec-default.js

```diff
@@ -54,7 +68,6 @@
           }
         };
 
-        pageAccelerator();
```

Comment:
Calling this method inside `verifyAjaxWasCalled` meant that the `isMetaKeyPressed` flag was getting reset after the simulated key event had been dispatched. Just pulled it up a level for now and all the tests pass.